src/RestServer/Controllers/v1/LedgerController.cs

```diff
@@ -0,0 +1,387 @@
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Witnesses);
+        }
+
+        /// <summary>
+        /// Gets the signers of a transaction.
+        /// </summary>
+        /// <param name="hash" example="0xad83d993ca2d9783ca86a000b39920c20508c8ccae7b7db11806646a4832bc50">Hash256</param>
+        /// <returns>An array of Signer object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/signers", Name = "GetTransactionSigners")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Signer[]))]
+        public IActionResult GetTransactionSigners(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Signers);
+        }
+
+        /// <summary>
+        /// Gets the transaction attributes of a transaction.
+        /// </summary>
+        /// <param name="hash" example="0xad83d993ca2d9783ca86a000b39920c20508c8ccae7b7db11806646a4832bc50">Hash256</param>
+        /// <returns>An array of the transaction attributes object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/attributes", Name = "GetTransactionAttributes")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(TransactionAttribute[]))]
+        public IActionResult GetTransactionAttributes(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Attributes);
+        }
+
+        #endregion
+
+        #region Memory Pool
+
+        /// <summary>
+        /// Gets memory pool.
+        /// </summary>
+        /// <param name="skip" example="1">Page</param>
+        /// <param name="take" example="50">Page Size.</param>
+        /// <returns>An array of the Transaction object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("memorypool", Name = "GetMemoryPoolTransactions")]
```

Comment:
Why `memorypool` is a part of `ledger`? I'd say it's a part of `node`, there is nothing here that is a part of the ledger yet.

Also, `ledger` can be associated with the respective native contract.

Reply:
`True`. Good point. I was thinking more that, `node` would be `settings`, `peers`, `connections`. However for `ledger` that would be, more like `transactions` and `blocks`; the actual data related the ledger as in; the stuff that keeps track of the accounts and data.